cfpb/cfgov-refresh
# Homepage: Make whole mobile carousel item a link

## Changes

- Make whole mobile carousel item a link instead of separate image and link.

## Testing

1. Pull branch and run server.
2. Check homepage and resize down to homepage size.
3. Entire carousel item should be clickable.
4. Compare to production, where image and link is only clickable.
